root/components/ExpandedArtistCredit.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
We have an `ExpandedArtistCredit` component at root/static/scripts/common/components/ExpandedArtistCredit.js which you should be able to use. (Might've been added since you wrote those.)